contracts/interfaces/0.8.x/IMintingAPICoreV0.sol

```diff
@@ -0,0 +1,54 @@
+// SPDX-License-Identifier: LGPL-3.0-only
+// Created By: Art Blocks Inc.
+
+pragma solidity ^0.8.0;
+
+interface IEngineContractMintingAPIV0 {
+    /**
+     * @notice Token ID `_tokenId` minted on project ID `_projectId` to `_to`.
+     */
+    event Mint(
+        address indexed _to,
+        uint256 indexed _tokenId,
+        uint256 indexed _projectId
+    );
+
+    // getter function of public variable
+    function nextProjectId() external view returns (uint256);
+
+    function projectDetails(uint256 _projectId)
+        public
+        view
+        returns (
+            string memory projectName,
+            string memory artist,
+            string memory description,
+            string memory website,
+            string memory license
+        );
+
+    function projectIdToPricePerTokenInWei(uint256 _projectId)
+        external
+        view
+        returns (uint256);
+
+    function projectTokenInfo(uint256 _projectId)
+        external
+        view
+        returns (
+            address,
+            uint256,
+            uint256,
+            uint256,
+            bool,
+            address,
+            uint256,
+            string memory,
+            address
+        );
```

Comment:
Makes sense that these are dependencies right now, but flagging as items that will need to be updated on minting API side when we upgrade Engine deployments to use V3 spec. cc @ryley-o

Reply:
Taking a look through the V3 spec right now 👍